packages/node_modules/@webex/plugin-meetings/src/meeting-info/collection.js

```diff
@@ -1,41 +1,36 @@
 import {find} from 'lodash';
 
-const MeetingInfoCollection = {
-  meetingInfos: {},
+import Collection from '../common/collection';
+import {MEETINGS} from '../constants';
+
+/**
+ * @class MeetingInfoCollection
+ */
+export default class MeetingInfoCollection extends Collection {
+  namespace = MEETINGS;
+
+  /**
+   * @memberof MeetingInfoCollection
+   * @constructor
+   * @public
+   */
+  constructor() {
+    super('meetingInfos');
+  }
 
   /**
-   *
    * @param {String} id ID of the meeting info you wish to retreive
    * @returns {MeetingInfo} returns a meeting info instance
+   * @public
+   * @memberof MeetingInfoCollection
    */
   get(id) {
```

Comment:
Would you use the `super`'s `get` here, or do I misunderstand?

Reply:
i am overriding the super's get here